ingest/clusters.py

```diff
@@ -116,7 +137,7 @@ def create_cluster_subdoc(
             "name": annot_name,
             "array_index": 0,
             "values": values,
-            "array_type": header_value_type,
+            "array_type": annot_type,
```

Comment:
Not a blocking concern, but calling this variable `annot_type` is somewhat misleading.  That makes me think of `group` or `numeric`, when in fact this value is either `cells`, `coordinates`, or `annotations`